Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM alpine:edge
 LABEL maintainer="dev@jpillora.com"
+
+ENV DNSMASQ_VERSION 2.82-r0
```

Comment:
You're using the package manager, you shouldn't need to do this. If you build this Dockerfile again, `edge` has been updated since the DockerHub build from 2 years ago, this will automatically use the newer package version, which is exactly what you've specified here.

You can either just trigger a new build of the Dockerfile without this PR and have a new release on DockerHub pushed, or build to the current Alpine stable 3.12 release in May, which provides dnsmasq 2.81, 2.82 will become available near the end of the year with Alpine 3.13.